SD_DB_Setup.sql

```diff
@@ -1,4 +1,5 @@
-drop database SisterDistrict_dev;
+SET SQL_MODE='ALLOW_INVALID_DATES';
+drop database if exists SisterDistrict_dev;
 
```

Comment:
Okay. So we can remove 'SET SQL_MODE='ALLOW_INVALID_DATES';' ? 

I can do that tonight. Sorry. I should have asked about that.